web/client/components/TOC/fragments/SettingsModal.jsx

```diff
@@ -153,7 +153,7 @@ const SettingsModal = React.createClass({
 
         if (this.props.settings.expanded) {
             return this.props.asModal ? (
-                <Modal {...this.props.options} show={this.props.settings.expanded} container={document.getElementById("body")}>
+                <Modal {...this.props.options} container={document.querySelector(".ms2 > #container") } show={this.props.settings.expanded}>
```

Comment:
cannot be fixed, at least it should be a property, better if it's not spread into every dialog. We should find a way to make it transparent. ms2 is overridable. #container is worse, this is a container, we cannot know the name, and we don't want to know it